interfaces/Glitter/templates/static/javascripts/glitter.main.js

```diff
@@ -388,6 +414,23 @@ function ViewModel() {
             self.isRestarting(1)
         }).always(self.setNextUpdate);
 
+        // this is a test of getting the entire queue
```

Comment:
Indeed, in this case I would **not** want a second API call but really for it to be moved to the top. I think the internal logic of the backend should only move it to the top of the priority of the job, so that a Normal priority will still be below High priority ones (if I remember cotrecly, should test that).